resources/assets/components/TagsFilter/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { map } from 'lodash';
+import classnames from 'classnames';
+
+import '../Tags/tags.scss';
+
+class TagsFilter extends React.Component {
```

Comment:
This should be a more general `MultiValueFilter` component. Then we can pass in what values it should render through the `props` we sent to it.